index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Personal portfolio</title>
+
+  <link rel="Stylesheet" href="./styles.css" />
+</head>
+
+
+<body>
+  <header class="main-header">
+    <a href="#" class="logo">Welcome</a>
+    <nav class="nav">
+      <ul class="nav-menu">
+        <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+        <li class="nav-link active"><a href="#About">About</a></li>
+        <li class="nav-link"><a href="#form">Contact</a></li>
+      </ul>
```

Comment:
- [ ] You should have a hamburger menu icon instead of the nav links. This links should be visible later in the desktop version of your project. 

![Captura de Pantalla 2022-09-23 a la(s) 7 23 06 a  m](https://user-images.githubusercontent.com/86022690/191959281-663908b8-b8f1-4ffb-b2ab-af8873ffa5a7.png)

Please do it as it is in the mobile version of your template 😄